homeassistant/helpers/service.py

```diff
@@ -358,8 +358,8 @@ def async_prepare_call_from_config(
                 f"Invalid config for calling service: {ex}"
             ) from ex
 
-    if CONF_SERVICE in config:
-        domain_service = config[CONF_SERVICE]
+    if CONF_ACTION in config:
+        domain_service = config[CONF_ACTION]
```

Comment:
Why don't we need to retain compatibility with `CONF_SERVICE` here, is it always rewritten by the `SERVICE_SCHEMA`?

Reply:
I'm not sure if I understand?

We want to move from `service` to `action` while maintaining backward compatibility, not making a forward compatibility by rewriting `action` as `service`?